src/vs/workbench/contrib/tasks/browser/runAutomaticTasks.ts

```diff
@@ -163,14 +163,15 @@ export class RunAutomaticTasks extends Disposable implements IWorkbenchContribut
 					label: nls.localize('allow', "Allow and run"),
 					run: () => {
 						resolve(true);
-						storageService.store(ARE_AUTOMATIC_TASKS_ALLOWED_IN_WORKSPACE, true, StorageScope.WORKSPACE, StorageTarget.MACHINE);
+						configurationService.updateValue(ALLOW_AUTOMATIC_TASKS, true, ConfigurationTarget.USER);
```

Comment:
I think these should go back to workspace so it's a per folder decision, not a single time per user.